paper/basic_training.tex

```diff
@@ -1098,8 +1098,8 @@ \subsubsection{Grid based Ewald summation}
 
 \item Transformation of the grid to reciprocal space: A Fast Fourier Transform (FFT) is used to convert the charges on the grid to their equivalent Fourier space structure factors.
 
-\item Energy calculation: The reciprocal space potential is calculated by solving the Poisson equation in Fourier space. At the same time, the grid is modified to store the reciprocal space potential. 
-\todo[inline, color={yellow!20}]{DLM: What does ``the grid is modified'' mean here and is it important? Does this mean just that the potential is stored at for each grid point?}
+\item Energy calculation: The reciprocal space potential is calculated by solving the Poisson equation in Fourier space. At the same time, the grid is replaced by the reciprocal space potential. 
```

Comment:
The next step refers to transforming the grid back, but the wording "the grid is replaced by" suggests that there is no grid anymore, so I don't think this edit works.

Check out what I've done in #76 and see if it works for you.

Reply:
Yes #76 is clearer. I am happy with that.